docs/apm-input-settings.asciidoc

```diff
@@ -22,25 +22,81 @@ OPTIONAL INFO AND EXAMPLE
 [id="input-{input-type}-host-setting"]
 `host`
```

Comment:
> The settings are not shown with these keys anymore.

Good point. It doesn't make sense to use these keys if the input config in Kibana doesn't use them.

> Maybe we do not even need to add them to the docs anymore?

I think we should talk about this. IMO, it still makes sense to have a configuration reference in the docs:

* There are some settings where everything a user needs to know isn't in the Kibana config—for example, that the first entry in the cipher suites config has the highest priority, and that if it's omitted, the Go crypto library’s default suites are used. Or how CORS works in relation to Access-Control-Allow-Headers.
* It makes the integration configs googleable. It could be someone interested in Elastic APM who wants to learn more about our config options without setting anything up, or someone frustrated as to why RUM isn't working that does a quick RUM config search.

I can change the titles to match the text in the Kibana, like `Origin Headers`, `Access-Control-Allow-Headers`, and `Custom HTTP response headers`, although some like `Curve types for ECDHE based cipher suites` might get a little long.

Reply:
makes sense!

Reply:
👍 Opened https://github.com/elastic/observability-docs/issues/1225 to track.